chart/helm-operator/values.yaml

```
@@ -42,6 +42,16 @@ tls:
   caContent: ""
   hostname: ""
 
+# ADVANCED: Allow for deploying Tiller as a sidecar (restricted to 'localhost' for security reasons).
+# When enabled, either .clusterRole.create should be true or .clusterRole.name should be set to the name of a cluster role granting the required privileges.
+# Please make extra sure you know what you are doing before enabling this. :)
+tillerSidecar:
+  enabled: false
+  image:
+    repository: gcr.io/kubernetes-helm/tiller
+    tag: v2.14.1
```

```suggestion
    tag: v2.14.3
```
Match https://github.com/fluxcd/helm-operator/blob/master/docker/helm.version